nuxeo-coldstorage-web/src/main/resources/web/nuxeo.war/ui/coldstorage/nuxeo-coldstorage-delete-blob-action.html

```diff
@@ -0,0 +1,41 @@
+<!-- 
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/) Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+`nuxeo-colstorage-delete-blob-action`
+@group Nuxeo UI
+@element nuxeo-colstorage-delete-blob-action 
+-->
+
+<script>
+  customElements.whenDefined('nuxeo-delete-blob-button').then(() => {
+    class ColdstorageDeleteBlob extends Nuxeo.DeleteBlobButton {
+      static get is() {
+        return 'nuxeo-coldstorage-delete-blob-action';
+      }
+
+      static get properties() {
+        return {
+          document: Object,
+        };
+      }
```

Comment:
I don't think we need this, right? The class we are extending already has `document` as a property.